### PR TITLE
Document how to get the account ID for account-scoped methods

### PR DESCRIPTION
**Pull request by weppos**
This was requested today from a customer. I realized we did not properly provide any instruction on how the account ID is supposed to be fetched.

### CODE REVIEW

File: content/v2.markdown [hunk 1]
```diff
@@ -118,6 +125,46 @@ All the timestamps and the dates are formatted according the [ISO-8601 standard]
...
+
+##### Getting the ID programmatically with an Account access token
+
+If you use an account token and you want to get the ID programmatically, the simplest way is to perform a [`whoami`](/v2/identity/#whoami) API call. The response will contain the account identifier as part of the `account` payload.
+
+~~~json
+<%= pretty_print_fixture("/whoami/success-account.http") %>
+~~~
+
+##### Getting the ID programmatically with an Account access token
```
Comment (3flex) [inline content/v2.markdown]: should be "with a User access token"?

**Comment by 3flex** [comment]
Small grammatical request - please use "a user" not "an user" throughout. If you say them both out loud the latter sounds odd. http://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

**Comment by weppos** [comment]
@3flex thanks for the link. As a non-native English speaker, this is something quite hard to remember to me as I'm used to the simplest approach _vowel-based_.

I agree that it should be _a user_.

**Comment by jacegu** [comment]
As I was going through this again I am wondering if we should have mentioned the `whoami` endpoint as a way of obtaining the `account_id`.

**Comment by weppos** [comment]
@jacegu I did, see https://developer.dnsimple.com/v2/#account-scope

> If you use an account token and you want to get the ID programmatically, the simplest way is to perform a whoami API call. The response will contain the account identifier as part of the account payload.

Do you have something different in mind?

**Comment by jacegu** [comment]
@weppos sorry, I missed that. I was expecting reading `whoami` in one the titles of the 3 ways of getting it and I totally over looked it. 😞

**Pull request merged**